tox.ini

```diff
@@ -55,8 +79,8 @@ commands =
     lint: flake8 --select FI14 --exclude 'h/cli/*,tests/h/cli/*,h/util/uri.py,h/migrations/versions/*' h tests
     format: black h tests
     checkformatting: black --check h tests
-    tests: coverage run -m pytest -Werror {posargs:tests/h/}
-    functests: pytest -Werror {posargs:tests/functional/}
+    tests: coverage run -m pytest {posargs:tests/h/}
+    functests: pytest {posargs:tests/functional/}
```

Comment:
The `-Werror` option is replaced by the `filterwarnings` setting above. Otherwise the command line option overrides the config file one